src/streams/StreamItem.js

```diff
@@ -47,19 +47,30 @@ const componentStyles = createStyleSheet({
   muted: {
     opacity: 0.5,
   },
+  collapseIcon: {
+    marginRight: 8,
+  },
+  touchable: {
+    // backgroundColor: 'green',
```

Comment:
Generally we avoid checking in commented-out code.

(I'm guessing this was here for debugging.)

Reply:
oops!